modules/3d/include/opencv2/3d/detail/submap.hpp

```diff
@@ -0,0 +1,544 @@
+// This file is part of OpenCV project.
+// It is subject to the license terms in the LICENSE file found in the top-level directory
+// of this distribution and at http://opencv.org/license.html
+
+#ifndef __OPENCV_RGBD_SUBMAP_HPP__
+#define __OPENCV_RGBD_SUBMAP_HPP__
+
+#include <opencv2/core/cvdef.h>
+
+#include <opencv2/core/affine.hpp>
+#include <type_traits>
+#include <vector>
+#include <map>
+#include <unordered_map>
+
+#include "opencv2/core/mat.inl.hpp"
+#include "opencv2/3d/detail/pose_graph.hpp"
+
+namespace cv
+{
+namespace detail
+{
+template<typename MatType>
+class Submap
+{
+   public:
```

Comment:
avoid 3 spaces indentation.

`public:` has 0 spaces indentation in core module.

Reply:
Fixed